1-exercises/H-array-methods-2/exercise3.js

```diff
@@ -7,7 +7,11 @@
 let ukNations = ["Scotland", "Wales", "England", "Northern Ireland"];
 
 function isInUK(country) {
-  return; // complete this statement
+  if (ukNations.includes(country)){
+     return true;
+  } else{
+     return false;
```

Comment:
Can just be

`return  ukNations.includes(country)`

Reply:
its a great idea i have just update it to return the result of the includes method directly, without an if...else statement. ^_^